.env.example

```
@@ -1,6 +1,16 @@
-ENV=prod
+# ENVIRONMENT ("development" or "production") is the variable read by
+# core/config.py and core/env.py. Defaults to "development" if unset.
+ENVIRONMENT=development
 LOGGER_LEVEL=INFO
 
+# OpenBB Platform API. Both dev and prod point at the same self-hosted,
+# production OpenBB instance - there is no local OpenBB by default. See
+# humblWORKSPACE's `pixi run dev:local-openbb` if you need to run OpenBB
+# locally instead (e.g. testing an OpenBB version bump).
+OPENBB_API_PROD_URL=https://data.humblfinance.io
+OPENBB_API_DEV_URL=https://data.humblfinance.io
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OPENBB_API_DEV_URL key should go before the OPENBB_API_PROD_URL key

(UnorderedKey)